src/main/java/li/cil/tis3d/api/ManualAPI.java

```diff
@@ -74,8 +64,7 @@ public static void addProvider(final ContentProvider provider) {
      * the image URL, i.e. <tt>![tooltip](prefix:data)</tt> will select the
      * image provider registered for the prefix <tt>prefix</tt>, and pass to
      * it the argument <tt>data</tt>, then use the returned renderer to draw
-     * an element in the place of the tag. The provided prefix is expected to
-     * be <em>without</em> the colon (<tt>:</tt>).
+     * an element in the place of the tag.
```

Comment:
This is a bit curious now, did the static entrypoint doc diverge from the interface one?

Reply:
I guess so. Probably even was/is in OC. Welp.